example/package.json

```diff
@@ -9,8 +9,8 @@
     "build:macos": "mkdirp dist && react-native bundle --entry-file index.js --platform macos --dev true --bundle-output dist/main.macos.jsbundle --assets-dest dist --reset-cache --config=metro.config.macos.js",
     "build:windows": "mkdirp dist && react-native bundle --entry-file index.js --platform windows --dev true --bundle-output dist/main.windows.bundle --assets-dest dist --reset-cache --config=metro.config.windows.js",
     "ci": "yarn --use-yarnrc .yarnrc-offline --prefer-offline",
-    "ios": "react-native run-ios --scheme ReactTestApp",
-    "macos": "react-native run-macos --scheme ReactTestApp",
+    "ios": "react-native run-ios",
+    "macos": "react-native run-macos --scheme Example",
```

Comment:
We don't need to specify the scheme anymore?

Reply:
Not since [0.2.15](https://github.com/microsoft/react-native-test-app/releases/tag/0.2.15).

Except for the macOS app. For some reason, rn-cli defaults to postfixing "-macOS" to the app name so it becomes "Example-macOS". Not sure whether to fix this upstream or here in our repo.

Reply:
Oh, cool. I didn't realize that was a side effect of the change I requested :D